client/src/views/SearchResults.tsx

```diff
@@ -0,0 +1,253 @@
+import Paper from '@mui/material/Paper';
+// import InputBase from '@mui/material/InputBase';
+import SearchIcon from '@mui/icons-material/Search';
+import { TextField } from '@mui/material';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+// import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+
+import type { Theme } from '@mui/material/styles';
+
+import AssetsList from './AssetsList';
+import OrgsList from './OrgsList';
+import VolunteerList from './VolunteerList';
+import FilterGroup from '../components/FilterGroup';
+import { filters1, filters2, filters3 } from '../assets/temp';
+import type { Asset } from '../types';
+import { APP_API_BASE_URL } from '../configs';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  searchBar: {
+    backgroundColor: 'white',
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    padding: '10px 5%',
+    borderRadius: 0,
+    alignItems: 'center',
+  },
+  iconButton: {
+    padding: 10,
+    '&:hover': {
+      backgroundColor: 'inherit',
+      borderRadius: '10px',
+    },
+  },
+  searchInput: {
+    display: 'flex',
+    alignItems: 'center',
+    paddingLeft: '10px',
+  },
+  contentWrapper: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+  },
+  leftPanel: {
+    borderRight: '1px solid grey',
+    width: '20%',
+  },
+  createBar: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'flex-end',
+  },
+  rightPanel: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-start',
+    padding: '10px 5%',
+    width: '80%',
+  },
+  makeAPost: {
+    fontWeight: 'bold',
+    textDecoration: 'none',
+    color: 'black',
+  },
+}));
+
+function SearchResults(): JSX.Element {
+  const classes = useStyles();
+  const location = useLocation<{ category: '' }>();
+  //implement below line to input filters at render
+  const querySearchCategory = location.state.category || '';
+  const querySearchText = queryString.parse(location.search).search;
+  const [searchCategory, setSearchCategory] = React.useState<string>(
+    (querySearchCategory as string) || '',
+  );
+  const [searchText, setSearchText] = React.useState<string>((querySearchText as string) || '');
+  const [inputText, setInputText] = React.useState<string>('');
+  const [selectedFilters, setSelectedFilters] = React.useState<{ [key: string]: boolean }>({});
+
+  const handleCheck = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSelectedFilters({
+      ...selectedFilters,
+      [event.target.value]: event.target.checked,
+    });
+  };
+
+  const [offers, setOffers] = React.useState<Asset[]>([]);
+  const [needs, setNeeds] = React.useState<Asset[]>([]);
+  const [orgs, setOrgs] = React.useState<Asset[]>([]);
```

Comment:
there's an `Organization` type you should grab from the types file to use here

Reply:
Removed VolunteerList for now until we create the OrgList Card/View and get designs. Put a placeholder in its place for now.

Reply:
still looks like it's using `Asset` type not `Organization`

Reply:
SearchResults.tsx
![Screenshot 2022-11-04 at 7 28 12 PM](https://user-images.githubusercontent.com/92830192/200097126-684faa68-69fd-4386-9d22-32e753a06a9d.png)

I was looking at commit [6fa2bbc](https://github.com/Nonprofit-Exchange-Hub/web-app/pull/230/commits/6fa2bbc334d5b86258dfed19363d727991761c32) 8 days ago. Let me know if you see something else and which commit it is. I'll fix it if so.